flang/docs/Directives.md

```diff
@@ -39,6 +39,8 @@ A list of non-standard directives supported by Flang
 * `!dir$ vector always` forces vectorization on the following loop regardless
   of cost model decisions. The loop must still be vectorizable.
   [This directive currently only works on plain do loops without labels].
+* `!dir$ ivdep` tells vecorization to ignore the dependency analysis of the
```

Comment:
Still missing a 'T'.

Reply:
Fixed... Sorry about the delay, I was hoping to have the rest of this work ready by today, but it's not there yet, and some other work is currently more important... :(